.travis.yml

```diff
@@ -13,4 +13,4 @@ install:
   - php composer.phar install
 
 script:
-  - phpunit -c tests/phpunit.xml
\ No newline at end of file
+  - php vendor/phpunit/phpunit/phpunit -c tests/phpunit.xml
```

Comment:
Try using `vendor/bin/phpunit` instead. Should actually be the same file but the path is more readable.